Checking

jjong0923 opened the pull request:
해결해야 될 문제 
1. 결과를 맞추고도 계속해서 DB에 맞췄을때의 카운트가 삽입됨
2. 결과를 맞췄을때 DB 목록이 뜸 -> 처음 시작할 때부터 DB 목록이 떠야 됨

1- > 결과를 맞추고 DB에 저장하면 바로 초기화 시키기 -> 맞춘 메세지 생략 됨 좀 더 생각해야 될 것 같음
2 -> DB에 3 ~ 4개 정도의 데이터를 미리 저장시키면 해결 될 것 같음

NuberBaseball Service에 변수들이 다 get이 필요해져서 Getter로 통합

jjong0923 committed:
add check list in controller

jjong0923 committed:
add check list in controller and unite Getter in service

Mutatio02 commented:
2-> schema.sql 에 더미 데이터를 삽입하고 Controller의 초기 페이지 경로에 해당 더미 데이터가 나오는지 확인 완료

Mutatio02 commented:
1->  게임 결과를 맞췄을 경우에 딱 한번 DB에 저장되도록 DB tryCount에 저장
기존에는 모델에 넘길 때 마다 isGameEnd() 호출되서 문제 -> flag변수를 둬서 관리하도록 변환

Mutatio02 committed:
fix the issues

1-> 저장 이슈 및 정답 맞췄을 떄 해당 문구가 나오도록 수정 및 DB저장 이슈 수정
2-> 처음에 DB에 더미 데이터를 넣고 맨 처음 화면에 보이도록 수정

jjong0923 committed:
add login.mustache and modify controller
